triangle/triangle.py

```diff
@@ -0,0 +1,79 @@
+import sys
```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 started bugfixing 
 Mistakes: 
 Dude...it is not even in right format.